src/test/resources/application.yml

```diff
@@ -41,3 +41,5 @@ collections:
   monitoring-locations:
     title: NWIS Monitoring Locations
     description: USGS water monitoring locations managed in the National Water Information System
+    contactName: ${MONLOC_CONTACT_NAME:""}
```

Comment:
Did the default not work? (I see the environment variable added to both the readme & the pom) If it works, it would be great to not have to add it to the pom or readme. If the default doesn't work, it should probably be removed from the application.yml(s)

Reply:
This environment variables resulted from acceptance criteria  "can be updated with YAML", which was taken to mean dynamic configuration.
@jkreft-usgs  Can the values just be hard coded or does this need to be dynamically 
configureable?

Reply:
The `monitoring-locations` collection contact information should be configurable from a config file (most likely YAML) on deploy.  I'm not knowledgable enough with spring to know what is appropriate in this test data.

Reply:
one possibility to avoid having the deploy configuration is to add a row in the network
table for monitoring-locations and then refactor the sql:
...
from nwis.network, nwis.network_sites, nwis.monitoring_location
where (network_code = 'monitoring-locations') or
  (network_code = #{collectionId} and
     nwis.network_sites.monitoring_location_id = nwis.monitoring_location.monitoring_location_id)

Reply:
My concern is having to add the value for automated testing. Line 44 and 45 are coded as if the value should be an empty string if not provided as an environment variable. It is ok to just hard code it here if the tests are expecting a specific value. If the default value does not work, it should be removed from the /src/main/resources/application.yml.
To be specific, I am talking about the use of the colon and default value within the ${}